multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/TimeoutAsyncFlowableStepWithHooks.java

```diff
@@ -27,18 +30,27 @@ public abstract class TimeoutAsyncFlowableStepWithHooks extends TimeoutAsyncFlow
 
     @Override
     public StepPhase executeAsyncStep(ProcessContext context) {
-        ModuleDeterminer moduleDeterminer = getModuleDeterminer(context);
         StepPhase currentStepPhase = context.getVariable(Variables.STEP_PHASE);
-        Module moduleToDeploy = moduleDeterminer.determineModuleToDeploy();
-        HooksCalculator hooksCalculator = getHooksCalculator(context);
-        HooksExecutor hooksExecutor = getHooksExecutor(hooksCalculator, moduleToDeploy);
-        List<Hook> executedHooks = hooksExecutor.executeBeforeStepHooks(currentStepPhase);
+        List<Hook> executedHooks = executeHooksForExecution(context, currentStepPhase);
+        context.setVariable(Variables.HOOKS_FOR_EXECUTION, executedHooks);
```

Comment:
Im a bit confused about the naming here - why are 'executedHooks' set as 'HOOKS_FOR_EXECUTION'?

Reply:
moved to execute hooks